x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/index.tsx

```diff
@@ -4,9 +4,53 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
```

Comment:
I think this file is not reachable, the fleet routing redirect to `/agents` by default

Reply:
yeah, that's ok for now as those changes are only in feature branch (this PR is against master); when these changes get integrated into the feature branch, I expect that we'll pull out the header code examples in this PR into their own files within each section